experimental/generator-dotnet-yeoman/generators/app/templates/webapp/runtime.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Again, put this once in the templates folder and copy it to the respective location for web app and Functions so that way we don't need to maintain multiple copies.